Bottom: 6e3c0c83ae162e914fd887cc05431b0bbd4cf65d
Top:    b931cea5dadd305b300bd7532d21a6603d85cc74
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-24 20:23:43 +0000

Refresh of add-a-demonstration-model-for-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 4703d7f..6780c9e 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -890,7 +890,7 @@ hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box)
     glStencilMask (stencil_bit);                            // Only write to our stencil bit
     glStencilFunc (GL_ALWAYS, stencil_bit, stencil_bit);    // Always pass stencil test, ref value is our bit
     glColorMask (0, 0, 0, 0);                               // Disable writting in color buffer
-    glDepthTest (GL_ALWAYS);
+    glDepthFunc (GL_ALWAYS);
     glDepthMask (GL_FALSE);
 
     /* It will already be setup like this (so avoid prodding the state-machine):
